engines/python/setup/djl_python/stable-diffusion.py

```diff
@@ -39,12 +40,10 @@ def __init__(self):
     def initialize(self, properties: dict):
         self.model_dir = properties.get("model_dir")
         self.model_id = properties.get("model_id")
-        self.data_type = properties.get("data_type", "fp32")
+        self.data_type = properties.get("data_type", "fp16")
```

Comment:
We should make it consistent with other handler:
1. data_type vs dtype, or we support both
2. default value should be None, we cast the data type only if user explicitly set the value

Reply:
good catch!